src/accountsdb/db.zig

```diff
@@ -570,33 +566,49 @@ pub const AccountsDB = struct {
                 defer accounts_file.close();
 
                 break :blk AccountFile.init(accounts_file, file_info, slot) catch |err| {
-                    self.logger.err().logf("failed to *open* AccountsFile {s}: {s}\n", .{ file_name_bounded.constSlice(), @errorName(err) });
+                    self.logger.err().logf("failed to *open* AccountsFile {s}: {s}\n", .{
+                        file_name_bounded.constSlice(),
+                        @errorName(err),
+                    });
                     return err;
                 };
             };
             errdefer accounts_file.deinit();
 
+            // index the account file
+            var slot_references = std.ArrayListUnmanaged(AccountRef).initBuffer(
+                references_buf[n_accounts_total..],
+            );
             indexAndValidateAccountFile(
                 &accounts_file,
                 self.account_index.pubkey_ref_map.shard_calculator,
                 shard_counts,
-                &references,
+                &slot_references,
                 geyser_slot_storage,
             ) catch |err| {
-                self.logger.err().logf("failed to *validate/index* AccountsFile: {d}.{d}: {s}\n", .{
-                    accounts_file.slot,
-                    accounts_file.id.toInt(),
-                    @errorName(err),
-                });
-                return err;
+                if (err == ValidateAccountFileError.OutOfReferenceMemory) {
+                    std.debug.panic("accounts-per-file-estimate too small ({d}), increase (using flag '-a') and try again...", .{accounts_per_file_est});
```

Comment:
Question: why `std.debug.panic` over `@panic`. In some places the former is used, while in other places the later.